roles/prosody_setup/tasks/main.yml

```diff
@@ -33,3 +33,41 @@
     owner: "{{ webapp_user }}"
     group: "{{ webapp_user }}"
     recurse: true
+
+- name: Ensure the presence of nfs-common
+  tags:
+    - setup
+    - nfs
+    - never
+  ansible.builtin.apt:
+    name: nfs-common
```

Comment:
this should be added to the list of default packages in the `build_dependencies` role